<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="backgroundwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{ 
background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, y2:1, stop:0 rgba(29, 30, 73, 255), stop:1 rgba(39, 41, 100, 255));}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>900</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Media/Des1UDT.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutProjectButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 20px;
	background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, 		y2:1, stop:0 rgba(0, 0, 0, 15), stop:1 rgba(0, 0, 0, 15));

	font: 70 12pt &quot;Century Gothic&quot;; color: rgba(54, 54, 54)
}

QPushButton:hover{
	background-color: rgba(73, 96, 96);
}
QPushButton:pressed{
	background-color: rgba(54, 54, 54);
}</string>
    </property>
    <property name="text">
     <string>About the
Project</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="howItWorksButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>560</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 20px;
	background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, 		y2:1, stop:0 rgba(0, 0, 0, 15), stop:1 rgba(0, 0, 0, 15));

	font: 70 12pt &quot;Century Gothic&quot;; color: rgba(54, 54, 54)
}

QPushButton:hover{
	background-color: rgba(73, 96, 96);
}
QPushButton:pressed{
	background-color: rgba(54, 54, 54);
}</string>
    </property>
    <property name="text">
     <string>How it
works?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startTrainingButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 20px;
	background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, 		y2:1, stop:0 rgba(0, 0, 0, 15), stop:1 rgba(0, 0, 0, 15));

	font: 70 18pt &quot;Century Gothic&quot;; color: rgba(54, 54, 54)
}

QPushButton:hover{
	background-color: rgba(73, 96, 96);
}
QPushButton:pressed{
	background-color: rgba(54, 54, 54);
}</string>
    </property>
    <property name="text">
     <string>Star
Training</string>
    </property>
   </widget>
   <widget class="QPushButton" name="visualizationButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 20px;
	background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, 		y2:1, stop:0 rgba(0, 0, 0, 15), stop:1 rgba(0, 0, 0, 15));

	font: 70 18pt &quot;Century Gothic&quot;; color: rgba(54, 54, 54)
}

QPushButton:hover{
	background-color: rgba(73, 96, 96);
}
QPushButton:pressed{
	background-color: rgba(54, 54, 54);
}</string>
    </property>
    <property name="text">
     <string>Visuals</string>
    </property>
   </widget>
   <widget class="QPushButton" name="predictionButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 20px;
	background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, 		y2:1, stop:0 rgba(0, 0, 0, 15), stop:1 rgba(0, 0, 0, 15));

	font: 70 18pt &quot;Century Gothic&quot;; color: rgba(54, 54, 54)
}

QPushButton:hover{
	background-color: rgba(73, 96, 96);
}
QPushButton:pressed{
	background-color: rgba(54, 54, 54);
}</string>
    </property>
    <property name="text">
     <string>Predict</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="buttonBGPhoto.qrc"/>
 </resources>
 <connections/>
</ui>
